apps/marketplace/components/ResetPassword/__snapshots__/ResetPassword.test.js.snap

```diff
@@ -29,7 +29,7 @@ exports[`Test suite for RequestResetEmailForm component Because snapshots 1`] =
         onSubmit={[Function]}
       >
         <div
-          className="field"
+          className="field "
```

Comment:
Do these snaps have to be updated?  Did we miss them before?

Reply:
The textfield component is updated